ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoConfigurationSpec.scala

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.store.dao
+
+import java.time.Instant
+
+import akka.stream.scaladsl.Sink
+import com.digitalasset.daml.lf.data.Ref
+import com.digitalasset.platform.store.entries.ConfigurationEntry
+import org.scalatest.{AsyncFlatSpec, Matchers}
+
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
+trait JdbcLedgerDaoConfigurationSpec { this: AsyncFlatSpec with Matchers with JdbcLedgerDaoSuite =>
```

Comment:
I am, in theory, fine with different ScalaTest styles, but can we pick one per package, at least?

Reply:
I'd rather not use `WordSpec` unless there is a clear need for it.